sdk/ai/azure-ai-projects/samples/agents/sample_agents_stream_eventhandler.py

```diff
@@ -84,8 +84,10 @@ def on_unhandled_event(self, event_type: str, event_data: Any) -> None:
     with project_client.agents.create_stream(
         thread_id=thread.id, assistant_id=agent.id, event_handler=MyEventHandler()
     ) as stream:
-        for _, _, custom_data in stream:
-            print(custom_data)
+        for event_type, event_data, func_return in stream:
+            print(
+                f"Streaming receive Event Type: {event_type}, Event Data in: {type(event_data)}, Event Function return: {func_return}"
+            )
```

Comment:
This is a long line... make sure you run `black --config ../../../eng/black-pyproject.toml .` in case it needs formatting

Reply:
Yes.   I already run black few times.